src/gui/MainWindow.cpp

```diff
@@ -412,6 +413,7 @@ MainWindow::MainWindow()
     m_ui->actionPasskeys->setIcon(icons()->icon("passkey"));
     m_ui->actionImportPasskey->setIcon(icons()->icon("document-import"));
     m_ui->actionEntryImportPasskey->setIcon(icons()->icon("document-import"));
+    m_ui->actionEntryRemovePasskey->setIcon(icons()->icon("document-close"));
```

Comment:
Would we want to use a new icon that is the passkey icon with an X on it?

Reply:
I could try to make one.

Reply:
The X is probably fine